.golangci.yml

```diff
@@ -65,34 +65,26 @@ linters-settings:
 linters:
   enable-all: true
   disable:
-    - deadcode
```

Comment:
Does this enable all the linters that are removed from this section? Or are there linters that were removed from golanci-lint and are then implicitly disabled?

Reply:
So, all of these were removed then?